File: SteadFastTimeSheet/ClientApp/src/app/models/timeSheet.ts
export interface Employee {
    Id: number;
    FirstName: string;
    LastName: string;
}

export interface TimeSheet {
    Id: number;
    EmployeeId: number;
    Date: Date;
    WorkHours: number;
    DateAdded: Date;
}

export interface EmployeeTimeSheet {
    employee: Employee;
    timeSheet: TimeSheet;
}

File: SteadFastTimeSheet/Models/TimeSheet.cs
using System;
using System.Collections.Generic;

namespace SteadFastTimeSheet.Models
{
    public partial class TimeSheet
    {
        public int Id { get; set; }
        public int EmployeeId { get; set; }
        public DateTime? Date { get; set; }
        public double? WorkHours { get; set; }
        public DateTime? DateAdded { get; set; }
    }
}
